app/views/miq_policy/_form.html.haml

```diff
@@ -27,10 +27,10 @@
             = _("Applies To")
           .col-md-8
             - if @policy.id
-              = MiqPolicy::POLICY_TOWHAT_APPLIES_TO_CLASSES[@edit[:new][:towhat]]
+              = MiqPolicy.policy_towhat_applies_to_classes(@edit[:new][:towhat])
```

Comment:
If we change the other side to just return the Hash always, and not take a parameter, then this just becomes simply:

```suggestion
              = MiqPolicy.policy_towhat_applies_to_classes[@edit[:new][:towhat]]
```